Added products/getPricing (#565)
MewsSystems/gitbook-connector-api: operations/README.md

| [Get all cancellation policies](cancellationpolicies.md#get-all-cancellation-policies) | Returns all cancellation policies filtered by services, rate groups and other filters |
| [Get all products](products.md#get-all-products) | Returns all products filtered by services or product identifier |
| [Delete products](products.md#delete-products) | Deletes specified products |

> | [Get product pricing](products.md#get-product-pricing) | Returns prices of a product in the specified interval |

Reviewer: ```suggestion
| [Get product pricing](products.md#get-product-pricing) | **Restricted!** Returns prices for a product for a specified time interval |
```
